### PR TITLE
Add small usage example for retrieving and using a UserInterface object

### PR DESCRIPTION
**Pull request by settermjd**
|    Q          |   A
|-------------- | ------
| Documentation | yes
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

I don't know if this PR is all that helpful, or stating the obvious, but I glossed over this when I was reading the documentation, so wanted to add it in in case others are doing the same.

### CODE REVIEW

File: docs/book/v1/intro.md [hunk 1]
```diff
@@ -50,6 +50,22 @@ if a user has been authenticated or not, e.g. it can be used to verify the
 authorization level of a user (for this scope, it is consumed by
 [mezzio-authorization](https://github.com/mezzio/mezzio-authorization)).
 
+For example:
+
+```
```
Comment (Ocramius) [inline docs/book/v1/intro.md]: @settermjd check CI feedback on this, please

**Review by Ocramius — CHANGES_REQUESTED** [review]

**Comment by froschdesign** [comment]
@settermjd 
> I don't know if this PR is all that helpful…

The entire documentation of the Mezzio's authentication components does not really help. There are no proper instructions on what to do or what to install at all. :disappointed:
Therefore, the addition is okay.

**Comment by froschdesign** [comment]
@settermjd 
> I don't know if this PR is all that helpful…

The entire documentation of the Mezzio's authentication components does not really help. There are no proper instructions on what to do or what to install at all. :disappointed:
Therefore, the addition is okay.

File: docs/book/v1/intro.md (hunk 1 above)
Comment (settermjd) [reply docs/book/v1/intro.md]: Shall do.

**Pull request merged**